explorer/pages/_error.tsx

```diff
@@ -1,5 +1,5 @@
+import { Error as CustomError } from "@clevercanary/data-explorer-ui/lib/components/Error/error";
```

Comment:
Do we need this change now or should this be done after the Error component is cleaned up?

Reply:
If we will just use the temp error component I'll just import it here instead of this one